modules/team/requestJoinCoreTeam.js

```diff
@@ -0,0 +1,10 @@
+const Promise = require('bluebird')
+const SendMail = require('../mail/mail')
+const { notificationEmail } = require('../mail/constants')
+
+module.exports = Promise.method(function requestJoinCoreTeam (params) {
+  SendMail.success(
+    { email: notificationEmail, language: 'en' },
```

Comment:
We need to create a new email notification here, case I'm receiving an error, cause the notification email is already as a cc.

So can you add to `constants` file a something like:
```
joinTeamEmail: "team@gitpay.me"
``` 
So you can import and use like this

```suggestion
    { email: joinTeamEmail, language: 'en' },
```